app/src/main/java/org/secuso/privacyfriendlytodolist/view/MainActivity.java

```diff
@@ -814,11 +820,12 @@ private void addListToNav() {
             MenuItem item = navMenu.add(R.id.drawer_group2, id, 1, name);
             item.setCheckable(true);
             item.setIcon(R.drawable.ic_label_black_24dp);
-            ImageButton v = new ImageButton(this, null, R.style.BorderlessButtonStyle);
-            v.setImageResource(R.drawable.ic_delete_black_24dp);
-            v.setOnClickListener(new OnCustomMenuItemClickListener(help.get(i).getId(), name, MainActivity.this));
-            item.setActionView(v);
-
+            if(!name.equals("default-list")){
```

Comment:
You should not assume the name will be "default-list". Compare using the id of the entry you saved earlier.